website/src/css/_navbar.scss

```
@@ -22,3 +22,29 @@
   padding: 5px;
   font-weight: 300;
 }
+/*===========*/
+
+/* Restyle gamburger button */
+
+.navbar__toggle {
+  width: 40px;
+  height: 40px;
+  background-color: var(--detox-color-a3);
+  border-radius: 10px;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Restyle search button */
+
+@media (max-width: 769px) {
```

Looks like you hypercorrected yourself and made a mistake involuntarily.
```suggestion
@media (max-width: 768px) {
```

All similar media queries in Docusaurus source code use `768px`, not `769px`.